net/tcp/tcp_send_buffered.c

```diff
@@ -1040,7 +1040,7 @@ ssize_t psock_tcp_send(FAR struct socket *psock, FAR const void *buf,
       goto errout;
     }
 
-  if (psock->s_type != SOCK_STREAM || !_SS_ISCONNECTED(psock->s_flags))
```

Comment:
Why `!_SS_ISCONNECTED(psock->s_flags)` is removed from condition here?

Reply:
just reference unbuffer send mode,  let us keep check here